android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMPBrowserActivity.java

```diff
@@ -71,10 +88,22 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         if (lowerURL.equals("about:blank")) {
           return true; // never load a blank page, e.g. when the component initializes
         }
-        if (FileUtils.isKeymanLink(lowerURL)) {
-          // KMAPro main activity will handle this intent
-          // Pass original url because path and query are case-sensitive
-          Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
+        Matcher matcher = pattern.matcher(url);
```

Comment:
To do: use standard url parser and match on path component, then extract bcp47 parameter with that.